doc/theatre.json

```diff
@@ -99,6 +99,114 @@
+                        "description": "Selects language of data you want to receive",
+                        "required": false,
+                        "type": "string",
+                        "default": "uk"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returned when all parameters were correct",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "page": {
+                                    "type": "integer"
+                                },
+                                "total_count": {
+                                    "type": "integer"
+                                },
+                                "count": {
+                                    "type": "integer"
+                                },
+                                "type": {
+                                    "type": "string"
+                                },
+                                "posts": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/Post"
+                                    }
+                                },
+                                "_links": {
+                                    "$ref": "#/definitions/PaginationLinks"
+                                }
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "Returned when the entities with given limit and offset are not found"
+                    }
+                }
+            }
+        },
+        "/post/{slug}": {
+            "get": {
+                "summary": "Find an Post by unique property {slug}",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "path",
+                        "name": "slug",
+                        "description": "Slug of Post to use",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "in": "query",
+                        "name": "locale",
+                        "description": "Selects language of data you want to receive",
+                        "required": false,
+                        "type": "string",
+                        "default": "uk"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returned when Post by {slug} was found",
+                        "schema": {
+                            "type": "object",
+                            "items": {
```

Comment:
object has properties not an items.
See your result - 

![Wrong swagger definition](http://storage7.static.itmages.com/i/17/0209/h_1486654568_8845064_f9edb48816.png)